gulpfile.js

```diff
@@ -50,6 +53,10 @@ let config = {
     }
 };
 
+console.log("");
+console.log("Building OpenEMR themes using Gulp".bold.yellow);
```

Comment:
this seem out of place on the output in the case of default install task since in that case is collecting packages and not building themes

Reply:
the yellow is nice though :)

Reply:
@bradymiller This gets run every time gulp is ran. What should it say instead?

Reply:
@tywrenn , Could use `if (config.install) {` to decipher on when to add build packages message versus build themes message (see bottom of script for how used on deciding the export to use)

Reply:
actually, i think you could even put the console.log output right there in that code block at the end of the script